src/main/java/org/prebid/server/spring/config/AnalyticsConfiguration.java

```diff
@@ -28,4 +42,98 @@ CompositeAnalyticsReporter compositeAnalyticsReporter(
     LogAnalyticsReporter logAnalyticsReporter(JacksonMapper mapper) {
         return new LogAnalyticsReporter(mapper);
     }
+
+    @Configuration
+    @ConditionalOnProperty(prefix = "analytics.pubstack", name = "enabled", havingValue = "true")
+    public static class PubstackAnalyticsConfiguration {
+
+        private static final String EVENT_REPORT_ENDPOINT_PATH = "/intake";
+
+        @Bean
+        PubstackAnalyticsReporter pubstackAnalyticsReporter(PubstackAnalyticsProperties pubstackAnalyticsProperties,
+                                                            HttpClient httpClient,
+                                                            JacksonMapper jacksonMapper,
+                                                            Vertx vertx) {
+            final org.prebid.server.analytics.pubstack.model.PubstackAnalyticsProperties properties =
+                    pubstackAnalyticsProperties.toComponentProperties();
+            final Map<EventType, PubstackEventHandler> eventHandlers = Arrays.stream(EventType.values())
```

Comment:
This initialization looks like a good candidate to move inside `PubstackAnalyticsReporter`. Are there solid reasons not to do this?